packages/http-client-csharp/generator/Microsoft.Generator.CSharp/src/Providers/NamedTypeSymbolProvider.cs

```diff
@@ -141,9 +141,16 @@ private void AddAdditionalModifiers(IMethodSymbol methodSymbol, ref MethodSignat
             var xmlDocumentation = propertySymbol.GetDocumentationCommentXml();
             if (!string.IsNullOrEmpty(xmlDocumentation))
             {
-                var xDocument = XDocument.Parse(xmlDocumentation);
-                var summaryElement = xDocument.Descendants(tag).FirstOrDefault();
-                return FormattableStringHelpers.FromString(summaryElement?.Value.Trim());
+                try
+                {
+                    var xDocument = XDocument.Parse(xmlDocumentation);
+                    var summaryElement = xDocument.Descendants(tag).FirstOrDefault();
+                    return FormattableStringHelpers.FromString(summaryElement?.Value.Trim());
+                }
+                catch (Exception)
```

Comment:
can we be more specific about how we get here?

Reply:
Updated to catch XmlException.